client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -267,6 +281,12 @@ class GSuiteAddUsers extends Component {
 						} ) }
 						noticeStatus="is-info"
 					>
+						{ selectedDomainName && (
+							<GoogleMailboxPricingNotice
+								domain={ selectedDomain }
+								googleMailProduct={ productType }
```

Comment:
This isn't an appropriate `ProductListItem`. I'll fix this so I can get this deployed today.